k2/addons/k2-sutevents/sutevents/__init__.py

```diff
@@ -0,0 +1,3 @@
+from zaf.messages.message import MessageId
+
+LOG_LINE_RECEIVED = MessageId('LOG_LINE_RECEIVED', """Log line received""")
```

Comment:
I can't remember exactly what sutevents did, but at least this `MessageId()` is needed to use the zserial component.

Reply:
If you happen to find k2-sutevents somewhere upstream, I'm happy to rebase again :smiley:

Reply:
I have a couple of more addons, including k2-sutevents.
I didn't think they where dependent, so I thought I would have a couple of days to fix them up. I'll commit them shortly.

Reply:
I've just merged #10 that contains k2-sutevents.
I'm sorry for the hassle of having to rebase again.